lib/matplotlib/tests/test_dates.py

```diff
@@ -449,6 +449,17 @@ def _create_auto_date_locator(date1, date2):
         assert list(map(str, mdates.num2date(locator()))) == expected
 
 
+def test_concise_formatter_subsecond():
+    locator = mdates.AutoDateLocator(interval_multiples=True)
+    formatter = mdates.ConciseDateFormatter(locator)
+    year_1996 = 9861.0
+    strings = formatter.format_ticks([
+        year_1996,
+        year_1996 + 500 / mdates.MUSECONDS_PER_DAY,
```

Comment:
Thanks for the test.  What happens if it is `year_1996 - 200 / us, year_1996, year_1996 + 200 / us`?  Also it would be good to know what the offset text says for these cases.

Reply:
with -200 +200 there are 2 different second values (59 and 00) so the original code compute level 5 correctly and you get ["59.9998", "00.0002"]